client/src/views/Chat.vue

```diff
@@ -6,29 +6,14 @@
         <div class="col-md-4 col-xl-3 chat">
           <div class="card mb-sm-3 mb-md-0 h-100 contacts_card">
             <div class="card-header text-light">
-              <p id="userId" class="inputBox">
-                User ID :
-                <strong @click="copy(user)" title="click to copy">{{
-                  user
-                }}</strong>
-              </p>
-              <ToastMessage :toastmsg="toastmsg" :toastType="toastType" />
-              <form v-on:submit.prevent="onSubmit" class="form">
-                <div class="input-group">
-                  <input
-                    type="text"
-                    class="form-control etrans"
-                    required
-                    v-model="room"
-                    placeholder="Other User's ID"
-                  />
-                  <div class="input-group-prepend">
-                    <button class="btn btn-dark" @click="join_room(room)">
-                      <strong>Add</strong>
-                    </button>
-                  </div>
-                </div>
-              </form>
+              <UserInfo 
+                :user="user" 
+                :toastmsg="toastmsg" 
+                :toastType="toastType" 
```

Comment:
Let's keep the toast separate from UserInfo component.

Reply:
Ok, I put it back in chat.vue, let me know if everything is ok now ;)